CHANGELOG.md

```diff
@@ -10,12 +10,350 @@ New features:
 
 Bugfixes:
 
-* Only include direct dependencies in the output for `purs graph` (#3993, @colinwahl)
+Other improvements:
 
-Fixes a bug where the transitive closure of a module's dependencies
-where included in the `depends` field in the output for `purs graph`.
+## v0.14.0
 
-Other improvements:
+### Polykinds
+
+Polymorphic kinds, based on the [Kind Inference for Datatypes](https://richarde.dev/papers/2020/kind-inference/kind-inference.pdf) paper (#3779, #3831, #3929, @natefaubion)
+
+Kinds are to types what types are to terms (except that kinds are actually themselves types now) but whereas we have polymorphic types, kinds were monorphic.
+
+This means that we were not able to abstract over kinds, leading for instance to a proliferation of proxy types:
+
+```purescript
+data Proxy (a :: Type) = Proxy
+```
+
+```purescript
+data SProxy (a :: Symbol) = SProxy
+```
+
+```purescript
+data RProxy (row :: # Type) = RProxy
+```
```

Comment:
Perhaps these could be merged together?
```suggestion
data Proxy (a :: Type) = Proxy
data SProxy (a :: Symbol) = SProxy
data RProxy (row :: # Type) = RProxy
data RLProxy (rowlist :: RowList) = RLProxy
```

Reply:
Do you think we should list all the proxies, including RLProxy as you suggested, BProxy and OProxy, or just the most commons?

Reply:
If our goal is to illustrate the idea, then these are likely enough. I don't think many people use `BProxy` or `OProxy` and if they do, then they are already familiar with type-level programming. Most people are probably familiar with `SProxy` and possibly with `RProxy`/`RLProxy` due to dealing with Record-related type class instances, so this list is likely enough.
If we want to be exhaustive, then adding those should be done.

Reply:
I added in RLProxy because seeing so many proxies helps drive the point home :) I don’t think it has to be exhaustive.

Reply:
I added the RowList proxy.